.devcontainer/devcontainer.json

```diff
@@ -15,8 +15,9 @@
     "pre-commit": "pre-commit install --config .devcontainer/.pre-commit-config.yaml"
   },
   "mounts": [
-    "source=automathor-cache,target=/home/vscode/.cache,type=volume",
-    "source=automathor-config,target=/home/vscode/.config,type=volume"
+    "source=partcad-partcad,target=/home/vscode/.partcad,type=volume",
+    "source=partcad-cache,target=/home/vscode/.cache,type=volume",
+    "source=partcad-config,target=/home/vscode/.config,type=volume"
```

Comment:
@clairbee here you can see that `partcad` does not follow common convention on where to store configurations and cache data. For example:

```
⬢ [Docker] ❯ tree -L 2 ~/.cache/ ~/.config/

/home/vscode/.cache/
├── ezdxf
│   └── font_manager_cache.json
├── ggshield
│   └── update_check.yaml
├── Microsoft
│   └── DeveloperTools
├── pip
│   ├── http-v2
│   └── selfcheck
└── starship

/home/vscode/.config/
├── ggshield
│   └── auth_config.yaml
├── starship.toml
└── vscode-dev-containers
    ├── conda-notice-already-displayed
    └── first-run-notice-already-displayed

12 directories, 6 files
```